Lab_works/Lab_works/Model/ApiManager.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import Alamofire
+import SwiftyJSON
+
+class ApiManager{
+    var articlesData: [ArticleModel] = []
```

Comment:
There is no reason to store it as instance variable. What your `fetchPhotosData` method does - it just loads stuff and gives it away, which means you should simply use local variable inside a method